client/src/components/landing/index.tsx

```diff
@@ -1,28 +1,54 @@
 import React, { ReactElement } from 'react';
 import { useTranslation } from 'react-i18next';
-
+import { useFeature } from '@growthbook/growthbook-react';
 import SEO from '../seo';
+import { Loader } from '../helpers';
 import AsSeenIn from './components/as-seen-in';
 import Certifications from './components/certifications';
 import LandingTop from './components/landing-top';
+import LandingTopB from './components/landing-top-b';
 import Testimonials from './components/testimonials';
 import Faq from './components/faq';
 
 import './landing.css';
 
+const LandingA = (): ReactElement => {
+  return (
+    <main className='landing-page'>
+      <LandingTop />
+      <AsSeenIn />
+      <Testimonials />
+      <Certifications />
+      <Faq />
+    </main>
+  );
+};
+
+const LandingB = (): ReactElement => {
+  return (
+    <main className='landing-page landing-page-b'>
+      <LandingTopB />
+      <Testimonials />
+      <Certifications />
+      <Faq />
+    </main>
+  );
+};
+
 function Landing(): ReactElement {
   const { t } = useTranslation();
+  const showLandingPageRedesign = useFeature('landing-page-redesign');
 
   return (
     <>
       <SEO title={t('metaTags:title')} />
-      <main className='landing-page'>
-        <LandingTop />
-        <AsSeenIn />
-        <Testimonials />
-        <Certifications />
-        <Faq />
-      </main>
+      {showLandingPageRedesign.value === true ? (
+        <LandingB />
+      ) : showLandingPageRedesign.value === false ? (
+        <LandingA />
+      ) : (
+        <Loader fullScreen={true} />
+      )}
```

Comment:
This doesn't work in development, since `value` is `null` meaning that we always end up showing the loader.

I'm not 100% sure, but my guess is the logic wants to be something like
```
  if (env.environment === 'development') return <LandingA />;
  if (feature.value === null) return <Loader fullScreen={true} />;
  if (feature.value) return <LandingB />;
  return <LandingA />;
  ```
  so that developers always see the old landing, but there's no ugly switch from A to B in production.

Reply:
I added a default set of features and re-arranged the logic. It should work now.